squad/ci/backend/lava.py

```diff
@@ -357,7 +357,7 @@ def __resubmit__(self, job_id):
         if self.use_xml_rpc:
             return self.proxy.scheduler.resubmit_job(job_id)
         response = requests.post(
-            urljoin(self.api_url_base, "jobs/%s/resubmit" % (job_id)),
+            urljoin(self.api_url_base, "jobs/%s/resubmit/" % (job_id)),
```

Comment:
While you're at it, could you please also add the trailing slash here https://github.com/Linaro/squad/blob/b2c17c7c333b37a4a95d33c9ce75fbdc96c98a42/squad/ci/backend/lava.py#L377

Reply:
I guess I can. This one isn't affected by redirect as it's GET not POST.